ee/clickhouse/views/events.py

```diff
@@ -100,7 +101,10 @@ def list(self, request: Request, *args: Any, **kwargs: Any) -> Response:
         team = self.team
         filter = Filter(request=request, team=self.team)
 
-        query_result = self._query_events_list(filter, team, request, limit=limit)
+        try:
+            query_result = self._query_events_list(filter, team, request, limit=limit)
+        except CHQueryErrorCannotCompileRegexp:
+            raise ValidationError("Invalid filter for events")
```

Comment:
Can we add a more detailed error here? Can the user do something to fix this?

Reply:
Yep!